Neo4j.Driver/Neo4j.Driver/Internal/Driver.cs

```diff
@@ -39,7 +39,7 @@ internal Driver(Uri uri, IConnectionProvider connectionProvider, IRetryLogic ret
             _retryLogic = retryLogic;
         }
 
-        public ISession Session(AccessMode defaultMode=AccessMode.Write, string bookmark = null)
+        public ISessionAsync Session(AccessMode defaultMode=AccessMode.Write, string bookmark = null)
```

Comment:
Shall we still have `ISession`, `ITransaction`, and `IStatementResult` as exposed types? I feel the surfix `Aync` is not very typical to used in interfaces. What do you think?